_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+    </tr>
+    <tr>
+        <td>Number of stakeholders</td>
+        <td>Thirteen individual stakeholders with distinct functional roles.</td>
+        <td>One cross-functional team focused on a product or project with all the required skillset to execute.</td>
+    </tr>
+    <tr>
+        <td>Compliance</td>
+        <td>Manual, tedious and error prone. Over burdens operational staff and causes a lot of waiting and toil for developers.</td>
+        <td>Automated with every build which reduces the risk of error. Frees up operational staff to focus on improvement and innovation. Developers have quick feedback.</td>
+    </tr>
+    <tr>
+        <td>Reporting and metrics</td>
+        <td>Manual collection of data, tedious and often inaccurate.</td>
+        <td>Automated data collection through telemetry from continuous integration and build pipelines. Wealth of information allows the creation of rich automated reports to identify issues and resolve quickly.</td>
+    </tr>
+
+ </tbody>
+</table>
+
+This drastically reduces lead time, we are no longer playing a game of catch up. We are able to put our focus on delivering the best possible services to our citizens rather than navigating a process. Developers are able to focus on their work of adding features and addressing technical debt. Operational staff are not over burdened by being constantly in fire fighting mode and can instead focus on innovating and improving the process. Management is able to quickly deliver and exceed their commitments. Risk is reduced by ensuring that every build of an application or service works as expected and is automatically compliant to our security and accessibility commitments that we are bound to by law.
+
+There is already some incredible work being done across IITB to move this forward. For example, did you know that you can get an Azure sandbox environment with large latitude to test out different tools and pipelines fairly easily? The Cloud Centre of Excellence ( CCoE ) and Specialized Environments (SE) teams in collaboration with have created a simple process to do so. I have gone through it myself took them less than half a day to provision and give me access! If this is something you need, you can find all the necessary information on their [sharepoint page](http://dialogue/grp/BU6518687/SitePages/CloudRequest.aspx) ( this is an internal link ). This is still a draft process and they are happy to accept any and all feedback to help them improve. As well as the CCOE and SE teams that are providing an awesome enablement capability there are also many development teams which have already adopted a DevOps way of working. ESDC Senior Advisors are working to drastically improve the release process in our organization as it is necessary to be able to respond to an increasingly changing environment. Continuing with our current process is a significant risk and we must work together as the technology organization to address this large organizational debt.
```

Comment:
I think there is a word missing in 'The Cloud Centre of Excellence (CCoE) and Specialized Environments (SE) teams in collaboration with ( ) have created....'

In the sentence 'I have gone through it myself took them less than...' should there be an 'and' between 'myself' and 'took'?

I think there should be a comma between 'capability' and 'there' in this sentence 'providing an awesome enablement capability there are also...'